Gopkg.lock

```diff
@@ -2,62 +2,47 @@
 
 
 [[projects]]
```

Comment:
Can you update your dep to latest version and rerun `./hack/update-vendor.sh`.
here's mine:
```console
installer git:(installconfig_in_old) ✗ dep version
dep:
 version     : v0.5.0
 build date  : 2018-07-26
 git hash    : 224a564
 go version  : go1.10.3
 go compiler : gc
 platform    : linux/amd64
 features    : ImportDuringSolve=false
```